error: lint `clippy::safe_fn` has been renamed to `clippy::safe_code`
   |
   = help: use the new name `clippy::safe_code`
   = note: requested on the command line with `-D clippy::safe_fn`
   = note: `-D renamed-and-removed-lints` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(renamed_and_removed_lints)]`

error: lint `clippy::safe_fn` has been renamed to `clippy::safe_code`
   |
   = help: use the new name `clippy::safe_code`
   = note: requested on the command line with `-D clippy::safe_fn`
   = note: duplicate diagnostic emitted due to `-Z deduplicate-diagnostics=no`

error: item must be `pub`
  --> tests/ui/manual_repeat_n.rs:5:1
   |
LL | fn main() {
   | ^ help: make this item public: `pub`
   |
   = note: requested on the command line with `-D clippy::missing-pub`

error: function must be `unsafe`
  --> tests/ui/manual_repeat_n.rs:5:1
   |
LL | fn main() {
   | ^
   |
help: make this function unsafe: `unsafe`
  --> tests/ui/manual_repeat_n.rs:5:1
   |
LL | fn main() {
   | ^
   = note: `-D clippy::safe-code` implied by `-D clippy::safe-fn`
   = help: to override `-D clippy::safe-fn` add `#[allow(clippy::safe_code)]`

error: item must be `pub`
  --> tests/ui/manual_repeat_n.rs:22:1
   |
LL | mod foo_lib {
   | ^ help: make this item public: `pub`

error: function must be `unsafe`
  --> tests/ui/manual_repeat_n.rs:23:9
   |
LL |     pub fn iter() -> std::iter::Take<std::iter::Repeat<&'static [u8]>> {
   |         ^
   |
help: make this function unsafe: `unsafe`
  --> tests/ui/manual_repeat_n.rs:23:9
   |
LL |     pub fn iter() -> std::iter::Take<std::iter::Repeat<&'static [u8]>> {
   |         ^

error: references are not allowed
  --> tests/ui/manual_repeat_n.rs:23:56
   |
LL |     pub fn iter() -> std::iter::Take<std::iter::Repeat<&'static [u8]>> {
   |                                                        ^^^^^^^^^^^^^
   |
   = help: use a raw pointer instead: `*const`
   = note: requested on the command line with `-D clippy::reference-used`

error: item must be `pub`
  --> tests/ui/manual_repeat_n.rs:28:1
   |
LL | fn foo() {
   | ^ help: make this item public: `pub`

error: function must be `unsafe`
  --> tests/ui/manual_repeat_n.rs:28:1
   |
LL | fn foo() {
   | ^
   |
help: make this function unsafe: `unsafe`
  --> tests/ui/manual_repeat_n.rs:28:1
   |
LL | fn foo() {
   | ^

error: lint `clippy::safe_fn` has been renamed to `clippy::safe_code`
   |
   = help: use the new name `clippy::safe_code`
   = note: requested on the command line with `-D clippy::safe_fn`
   = note: duplicate diagnostic emitted due to `-Z deduplicate-diagnostics=no`

error: missing `#![no_std]`
   |
   = help: make this crate `#![no_std]`
   = note: requested on the command line with `-D clippy::missing-no-std`

error: aborting due to 11 previous errors

